features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NRF51/source/nRF5xGap.cpp

```diff
@@ -259,12 +260,10 @@ ble_error_t nRF5xGap::startAdvertising(const GapAdvertisingParams &params)
     whitelist.addr_count = 0;
     whitelist.irk_count  = 0;
 
-    /* Add missing IRKs to whitelist from the bond table held by the SoftDevice */
-    if (advertisingPolicyMode != Gap::ADV_POLICY_IGNORE_WHITELIST) {
```

Comment:
has this policy been removed?

Reply:
No it hasn't been removed and it is still used line 299.